portality/forms/application_processors.py

```diff
@@ -373,7 +376,12 @@ def finalise(self, account, save_target=True, email_alert=True):
                 sent = False
                 send_report = []
                 try:
-                    send_report = emails.send_publisher_reject_email(self.target, update_request=is_update_request)
+                    if self.source.application_status != self.target.application_status:
+                        eventsSvc.trigger(models.Event(constants.EVENT_APPLICATION_STATUS, account.id, {
+                            "application": self.target.data,
+                            "old_status": self.source.application_status,
+                            "new_status": self.target.application_status
+                        }))
```

Comment:
This trigger isn't required here, as the application status change event is triggered at the end of the function anyway